sdk/python/foundation-models/llama2/promptFlow/Readme.md

```diff
@@ -0,0 +1 @@
+These are the samples of Prompt flows.
```

Comment:
Explain here a bit more which are the two examples about. Use proper README formats with headers. Don't need to add much, it can be an small table or bullets with an small description and link to README file on each example (if any).

Reply:
Done, I explained briefly.